wizeline/remix-project-lab
# feat/203/Update homepage

# What does this PR do?

Update our current homepage. adds new section called Experience in /profile, this section help collect info to show it in the homepage

# Where should the reviewer start?

app/routes/index.tsx
app/routes/profile/$email/index.tsx

# How should this be manually tested?

Go to /profile
add experience at the bottom
Go to /
Check if your experience is there

# Any background context you want to provide?

# What are the relevant tickets?

https://github.com/wizeline/remix-project-lab/issues/203

# Screenshots

![Captura de pantalla 2023-08-24 a la(s) 10 58 50](https://github.com/wizeline/remix-project-lab/assets/93209499/9366bbd8-1a1d-4e58-ba4c-88d185784bca)

![Captura de pantalla 2023-08-24 a la(s) 10 59 32](https://github.com/wizeline/remix-project-lab/assets/93209499/9366a311-b095-4b4d-8842-7e5e22e267c4)

![Captura de pantalla 2023-08-24 a la(s) 10 59 54](https://github.com/wizeline/remix-project-lab/assets/93209499/90583cf9-89ba-46d5-998a-64aaaaec89d5)


# Questions

# Checklist
